WORKSPACE

```diff
@@ -372,16 +372,16 @@ skydoc_repositories()
 
 http_archive(
     name = "io_bazel_rules_go",
-    sha256 = "9084496dde809363c491137e077ace81780463ead0060a0a6c3c4c0f613e9fcb",
-    strip_prefix = "rules_go-0.18.6",
-    urls = ["https://github.com/bazelbuild/rules_go/archive/0.18.6.tar.gz"],
+    sha256 = "1ad10f384053ae50c050fdec7d595d12427c82c0a27c58f8554deb4437216892",
+    strip_prefix = "rules_go-0.18.7",
+    urls = ["https://github.com/bazelbuild/rules_go/archive/0.18.7.tar.gz"],
 )
 
 http_archive(
     name = "com_github_bazelbuild_buildtools",
-    sha256 = "86592d703ecbe0c5cbb5139333a63268cf58d7efd2c459c8be8e69e77d135e29",
-    strip_prefix = "buildtools-0.26.0",
-    urls = ["https://github.com/bazelbuild/buildtools/archive/0.26.0.tar.gz"],
+    sha256 = "81fa031663fe67eab14b6b20131e5af1155f3e60b58da71249a3c744903710cd",
+    strip_prefix = "buildtools-0ce08111cb60d0f284a4399a36e23c28a934558a",
```

Comment:
buildtools 0.28 just came out: https://github.com/bazelbuild/buildtools/releases/tag/0.28.0. Is that one sufficient?

Reply:
Hey, good point.

Reply:
Done.